model_compiler/python/onnx_to_nnir.py

```diff
@@ -164,23 +171,26 @@ def onnx2ir(model, output_folder):
     else:
         raise TypeError("Model must be file path to .onnx file or onnx loaded model")
     # convert ONNX graph to IR graph and save
-    graph = onnx_graph_to_ir_graph(onnx_model.graph)
+    graph = onnx_graph_to_ir_graph(onnx_model.graph, input_dims, output_dims)
     graph.toFile(output_folder)
 
 def main():
-    if len(sys.argv) < 3:
-        print('Usage: python onnx2nnir.py <onnxModel> <nnirOutputFolder>')
+    if len(sys.argv) < 7:
+        print('Usage: python onnx2nnir.py <onnxModel> <nnirOutputFolder> --input-dims n,c,h,w --output-dims n,c,h,w')
```

Comment:
The input and output dims need to be optional and work by default like before. The new options need to be documented in the readme.

Reply:
I have made the input and output dimensions to be optional and added it to the readme.

Reply:
Remove `--output-dims` option from the script. The script needs to use the `n' from the input, (c,h,w) needs to come from the model. Talk to @rrawther if you have any questions.